include/cudeconvolve.h

```diff
@@ -1,7 +1,7 @@
-#ifndef __DECONVOLVE_H__
```

Comment:
This header isn't part of the user-facing API, right? Maybe it can stay in `src/`? I don't know what a good convention is here.

Reply:
I agree (see commit 5ab18e4).